pkg/operator/events/recorder_upstream.go

```diff
@@ -27,20 +31,63 @@ type upstreamRecorder struct {
 	broadcaster       record.EventBroadcaster
 	eventRecorder     record.EventRecorder
 	involvedObjectRef *corev1.ObjectReference
+	options           record.CorrelatorOptions
+
+	// shuttingDown indicates that the broadcaster for this recorder is being shut down
+	shuttingDown  bool
+	shutdownMutex sync.RWMutex
+
+	// fallbackRecorder is used when the kube recorder is shutting down
+	// in that case we create the events directly.
+	fallbackRecorder Recorder
+}
+
+var DefaultOperatorEventRecorderOptions = record.CorrelatorOptions{
+	BurstSize: 60,       // default: 25 (change allows a single source to send 50 events about object per minute)
+	QPS:       1. / 60., // default: 1/300 (change allows refill rate to 1 new event every 2 minutes)
```

Comment:
I think is just `1.0` for one per second.  We want them all!

Reply:
done